src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "string",
+             "default": null,
+             "description": "Channel name. If none, try to auto detect channel",
+             "name": "channel_name"
```

Comment:
@edersondisouza 
I suggest to @yongli3  this method is for easy for user to use the channel that is not supported yet. Can we add custom iio node (not tie to any sensor type) that can let user specify this option.

I got another scenario to discuss on how to handle in iio flow node.
sx9500 sensor have more channel(in_proximity0_raw, in_proximity1_raw, in_proximity2_raw, in_proximity3_raw), how can user select 0 and 2 only in json?